src/core/scene/a-scene.js

```diff
@@ -312,6 +343,11 @@ module.exports = registerElement('a-scene', {
         renderer.setPixelRatio(window.devicePixelRatio);
         renderer.sortObjects = false;
         this.effect = new THREE.VREffect(renderer);
+        this.effect.autoSubmitFrame = false;
+
+        // Create a render target for post processing.
+        this.renderTarget = new THREE.WebGLRenderTarget(1, 1, { minFilter: THREE.LinearFilter, magFilter: THREE.NearestFilter, format: THREE.RGBAFormat });
+        this.renderTarget.texture.generateMipmaps = false;
```

Comment:
Seems like this could be be lazily instantiated either here or on line L484 only when `this.getAttribute('postprocessing')` is set to `true` instead of always instantiated right here.

Seems inefficient unless this is a scoping/ease-of-use thing because `this.renderTarget` is never used otherwise.